app/femr/ui/views/medical/listVitals.scala.html

```diff
@@ -41,11 +41,10 @@
         <tr id="temperature">
             <th>T</th>
             @for(dateIndex <- 1 to vitalMap.getDateListChronological.size) {
-                <td>@outputTemperatureOrNA(
-                        vitalMap.get("temperature", vitalMap.getDate(dateIndex - 1)),
-                        viewModel.getSettings.isMetric,
-                        ""
-                )</td>
+                <td>
+                     @outputTemperatureOrNA(vitalMap.get("temperatureFahernheit", vitalMap.getDate(dateIndex - 1)), viewModel.getSettings.isMetric, "")
+                    /@outputTemperatureOrNA(vitalMap.get("temperatureCelsius", vitalMap.getDate(dateIndex - 1)), !viewModel.getSettings.isMetric, "")
```

Comment:
If I turn on the metric system option, then temperatureFahrenheit stores the celsius value